tests/scaffoldings/dotd/custom/entrypoint.d/600-goodconf.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/local/bin/python-odoo-shell
+from odoo.tools import config
+assert config.get("limit_request") == 9000
```

Comment:
I'm afraid we will need some more magic to make this test pass, because [on v11 you cannot have duplicate config entries](https://travis-ci.org/Tecnativa/docker-odoo-base/jobs/316530702#L3757), 😕 possibly a new limitation on Python 3 implementation.

Merging as is would break v11 production deployments, not nice really... (Blessed be tests!)

Reply:
Well I think we just need another configuration variable. I've now exhausted all the ones that exist in Odoo, so I'll just make one up I think